shared.props

```diff
@@ -1,4 +1,5 @@
 <Project>
+  <Sdk Name="DotNet.ReproducibleBuilds.Isolated" Version="1.2.25" />
```

Comment:
I tried to use the variable instead of duplicating the version number, but encountered `error MSB4236: The SDK 'DotNet.ReproducibleBuilds.Isolated' specified could not be found.` when I tried to use `$(ReproducibleBuildsVersion)` here, and moving the reference to versions.props above this line didn't make a difference

Reply:
See https://learn.microsoft.com/en-us/visualstudio/msbuild/how-to-use-project-sdk?view=vs-2022. Have you tried importing `versions.props` above this line?

Reply:
When I saw that project, I was reminded of the last couple rounds of new build/package features being a royal PITA to get working and thought it would be nice to remove an item from our pile of maintenance responsibilities... But you are right about it not really helping us that much today, especially this SDK bit. I'll take the refs back out and we'll keep managing it here, and we'll have this exchange as a record for where to look in the future if things change

Reply:
> Something is still not right with this suggestion:
> 
> > The only thing we truly need is embedded to replace IncludeSymbols and SymbolPackageFormat.
> 
> The build still produced .snupkg files, which I think was the reason you can't step into packages from the devops feed. But some combination of switches I tried yesterday did work, so I'll keep at it

Just kidding, I was apparently looking at the wrong build